parlai/zoo/model_list.py

```diff
@@ -3,10 +3,12 @@
 # This source code is licensed under the BSD-style license found in the
 # LICENSE file in the root directory of this source tree. An additional grant
 # of patent rights can be found in the PATENTS file in the same directory.
-"""This file contains a list of all the models in the model zoo,
-the path to load them, agents & tasks associated (e.g. they were trained using) and a description.
-Using the path you should be able to download and use the model automatically, e.g.:
- python examples/interactive.lua --model-file "models:wikipedia_2016-12-21/tfidf_retriever/drqa_docs"
+"""This file contains a list of all the models in the model zoo, the path to
+load them, agents & tasks associated (e.g. they were trained using) and a
+description. Using the path you should be able to download and use the model
+automatically, e.g.:
+ python examples/interactive.lua --model-file\
```

Comment:
just realized this says `interactive.lua`, could you possibly change to `interactive.py`